core/src/chain/mod.rs

```diff
@@ -149,7 +151,7 @@ impl<T: HashTable> Chain<T> {
     }
 
     /// get a Pair by Pair/Header key from the HashTable if it exists
-    pub fn get(&self, k: &str) -> Result<Option<Pair>, HolochainError> {
+    pub fn get_pair(&self, k: &str) -> Result<Option<Pair>, HolochainError> {
```

Comment:
could change this to just `pair()` as per our discussions on #243

Reply:
yes!